ironfish-zkp/src/circuits/util.rs

```diff
@@ -132,3 +140,34 @@ where
 
     Ok(value_bits)
 }
+
+/// Computes the note commitment, returning the full point.
+pub fn commitment_full_point(note: Note) -> jubjub::SubgroupPoint {
```

Comment:
`circuits/util.rs` is for functions used in circuits. While this is used in a test for a circuit, it isn't being used by a circuit directly. What do you think about putting this in like `ironfish-zkp/src/util.rs` or something?